fetch-jars

```diff
@@ -26,8 +27,8 @@ function has_jars {
 rev=""
 
 # Look for jars in the specified revision
-if [[ -n $REV_NAME ]]; then
-  rev=$(git rev-parse "$REV_NAME")
+if [[ -n $rev_name ]]; then
```

Comment:
doesn't this fail shellcheck? if I call `./fetch-jars "x || 1 = 1"` does that break things?

Reply:
shellcheck isn't complaining.  As I would expect that fails with
```
fatal: ambiguous argument 'x && 0 = 1': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Reply:
Oh, weird, first time I've ever experienced shellcheck not complaining _enough_ about a variable without `"`'s around it lol

idk, if we care, put `"`s around the vars; if not, I guess a user who puts spaces in a ref name is kinda asking for trouble anyway...